### ISSUE TITLE
Resend User Confirmation Email - PHP issue in 7.4.0

### ISSUE THREAD

**Issue by SMA-cgn** [in2code-de/femanager#596]
Hi,

when clicking the Resend User Confirmation Email button I get following error message.

![image](https://github.com/user-attachments/assets/f35f73d4-b2d5-4d8b-b554-987fc8f1b994)

![image](https://github.com/user-attachments/assets/441d8de7-517d-4367-9032-5451c508d27c)

Core: Exception handler (WEB): Uncaught TYPO3 Exception: Call to a member function sL() on null | Error thrown in file /opt/bitnami/typo3/public/typo3/sysext/frontend/Classes/ContentObject/ContentObjectRenderer.php in line 4428. Requested URL: https://.../typo3/module/web/FemanagerM1?token=--AnonymizedToken--&id=2&tx_femanager_web_femanagerm1%5BuserIdentifier%5D=132&tx_femanager_web_femanagerm1%5Baction%5D=resendUserConfirmationRequest&tx_femanager_web_femanagerm1%5Bcontroller%5D=UserBackend

When clicking the "Decline User" button I get following error message:

![image](https://github.com/user-attachments/assets/5c2ab9d9-8488-45e3-a06c-dc722fa29463)

Core: Exception handler (WEB): Uncaught TYPO3 Exception: Class "In2code\Femanager\Controller\RequestFactory" not found | Error thrown in file /opt/bitnami/typo3/public/typo3/sysext/core/Classes/Utility/GeneralUtility.php in line 3215. Requested URL: https://.../typo3/module/web/FemanagerM1?token=--AnonymizedToken--&id=2&tx_femanager_web_femanagerm1%5BuserIdentifier%5D=132&tx_femanager_web_femanagerm1%5Baction%5D=refuseUser&tx_femanager_web_femanagerm1%5Bcontroller%5D=UserBackend

I have following environment:

TYPO3 Version	11.5.37
Webserver	Apache
PHP Version	8.0.19
Database (Default)	MySQL 10.6.7-MariaDB
Application Context	Production
Composer mode	Enabled
Operating System	Linux 4.19.0-27-cloud-amd64

Thanks a lot for your support!

**Comment by sbusemann** [comment]
Which version of femanager do you use?

**Comment by SMA-cgn** [comment]
We are using 
femanager | 7.4.0

Thanks!

**Comment by sbusemann** [comment]
Please take care, that 

Backend Configuration
^^^^^^^^^^^^^^^^^^^^^

Take care, that the setting "module.tx_femanager.settings.configPID" is set up for any site (via Page TS or User TS or in the TypoScript Setup)

You need to specify the pid, where your main typoscript settings (frontend) is stored. You should take a page, where a plugin (regristration / invitation) is available.. This is needed to perform actions like confirm users via Backend or resend

**Comment by SMA-cgn** [comment]
This setting is already in place:

![image](https://github.com/user-attachments/assets/8ee166af-c3b3-4283-8717-6c4c1eb6130b)

When adding module.tx_femanager.settings.configPID = 1 to the typoscript setup the error message does not come up anymore when clicking the Resend User Confirmation Email. When clicking it the view changes to the List overview. However, no mail is sent to the user again. What would be the expected outcome?

Is it necessary to have the resend confirmation email plugin in the FE?

**Comment by sbusemann** [comment]
The Page ID 1 then needs to contain a Femanager plugin. You can set the pid to anyother page, which contains a femanager plugin. This is needed, at the FE is used to generate the mail and links ...

**Comment by SMA-cgn** [comment]
Thanks for your support!!!

The femanager plugin was already added to PID 1.
I added a new subpage, added the femanager there and updated the configPID. However, I still get the same error messages above. The view does not switch back to the list view as described above. This only happens when the Frontend Users module is opened on a pagetree item, that is not the users folder.

Which view is required? I tested it with Registration & Resend User Confirmation.

What is this constant setting for? Is it required to make the backend function work?
plugin.tx_femanager.settings.showResendUserConfirmationRequestView
